borg/archiver.py

```diff
@@ -1399,7 +1440,7 @@ def build_parser(self, args=None, prog=None):
         subparser.add_argument('location', metavar='ARCHIVE',
                                type=location_validator(archive=True),
                                help='name of archive to create (must be also a valid directory name)')
-        subparser.add_argument('paths', metavar='PATH', nargs='+', type=str,
+        subparser.add_argument('paths', metavar='PATH', nargs='*', type=str,
```

Comment:
I'm not so sure that writing things like `repo --options paths` is as uncommon as you think. I have for example... `$create::hostname-$date $OPTS -e ... /some /paths` in a script, that / similar patterns are perhaps not as uncommon.